scripts/run-wsdunit

```diff
@@ -13,27 +13,41 @@ if [ -z "$name" ]; then
     echo "missing parameter: test name"
     exit 1
 fi
+shift # Eat the first argument.
 
-limit="$2"
-count=0
-while true
-do
-    count=$((count+1))
-
-    (cd test; ./run_unit.sh --test-name $name.la --log-file $name.log --trs-file $name.trs --color-tests yes --enable-hard-errors yes --expect-failure no -- ./$name.la)
-    ./coolwsd --disable-cool-user-checking --cleanup
-    if ! grep -q FAIL test/$name.trs; then
-        RET=0
-        break
```

Comment:
This prevented finding out if the test passed sometimes or never.

Reply:
Heh, it was intentional, but I can probably live with the new behavior as well. Funny, now that I re-read the comment at the top of the file, indeed it describes what you did.